app/javascript/dashboard/routes/dashboard/settings/reports/LabelReports.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <woot-reports
+    key="label-reports"
+    type="label"
+    getter-key="labels/getLabels"
+    action-key="labels/get"
+    :download-button-label="$t('REPORT.DOWNLOAD_AGENT_REPORTS')"
```

Comment:
Button text should be "Download label reports"

Reply:
Fixed in

https://github.com/chatwoot/chatwoot/pull/3084/commits/26ee689277da85b14e4706210103aef7bcb51d06